bindings/azure/storagequeues/storagequeues.go

```diff
@@ -178,8 +184,15 @@ func (d *AzureQueueHelper) Read(ctx context.Context, consumer *consumer) error {
 	}
 
 	_, err = consumer.callback(ctx, &bindings.ReadResponse{
-		Data:     data,
-		Metadata: map[string]string{},
+		Data: data,
+		Metadata: map[string]string{
+			messageId:       *res.Messages[0].MessageID,
+			insertionTime:   res.Messages[0].InsertionTime.Format("2006-01-02 15:04:05"),
```

Comment:
Please use `time.RFC3339` which is the standard time format we use across Dapr. Otherwise, LGTM

Reply:
Done